app/routes/errors.py

```diff
@@ -34,8 +32,14 @@ def forbidden(error=None):
 @errors_blueprint.app_errorhandler(SubmissionFailedException)
 @errors_blueprint.app_errorhandler(Exception)
 def internal_server_error(error=None):
-    log_exception(error, 500)
-    return _render_error_page(500)
+    try:
+        log_exception(error, 500)
+        return _render_error_page(500)
+    except Exception:  # pylint:disable=broad-except
+        logger.error(
+            'an error has occurred', exc_info=True, url=request.url, status_code=500
```

Comment:
This error message should be different to the other generic one, maybe something like "error when rendering 500 error".

Reply:
👌